react/components/organisms/chat/chat.js

```diff
@@ -54,6 +54,8 @@ export class Chat extends PureComponent {
     onRichTextInputFocus = () => {
         this.scrollToEnd();
     };
+    
+    richTextInputBlur = () => this.richTextInputComponent.blur();

```

Comment:
This function is duplicated. Call it `blur` and move it to be before the callbacks (set of `onX`).

Also, add a `focus` method too.